constants/ffmpeg.go

```diff
@@ -3,15 +3,19 @@ package constants
 type FILE_TYPE int
 
 const (
-	Audio192K FILE_TYPE = iota
-	ImagePng
+	ImagePng FILE_TYPE = iota
 	Video5M
 	Video3M
 	Video1M
 	Video800K
 	Video400K
 )
 
+const (
```

Comment:
lets keep this const in the upper one, else iota value starts at 0, so ImagePng and Audio192K will have same value